src/util/appConfigHelper.js

```diff
@@ -1,46 +1,17 @@
 /* istanbul ignore file */
 
-import store from 'redux/store';
-import {
-  setData as setDataInRedux,
-  removeData as removeDataFromRedux
-} from 'redux/appConfig';
-
-const isLocalStorageAvailable = () => {
-  try {
-    localStorage.setItem('CLEENG_LS', 'yes');
-    if (localStorage.getItem('CLEENG_LS') === 'yes') {
-      localStorage.removeItem('CLEENG_LS');
-      return true;
-    }
-    return false;
-  } catch (e) {
-    return false;
-  }
-};
-
 export const getData = name => {
-  const result = isLocalStorageAvailable()
-    ? localStorage.getItem(name)
-    : store.getState().appConfig[name];
+  const result = localStorage.getItem(name);
   if (!result && name === 'CLEENG_AUTH_TOKEN') {
-    console.error(
-      `Unable to get CLEENG_AUTH_TOKEN from local storage or redux store`
-    );
+    console.error(`Unable to get CLEENG_AUTH_TOKEN from local storage`);
     return null;
   }
   return result;
 };
 
-export const setData = (name, value) =>
-  isLocalStorageAvailable()
-    ? localStorage.setItem(name, value)
-    : store.dispatch(setDataInRedux({ name, value }));
+export const setData = (name, value) => localStorage.setItem(name, value);
 
-export const removeData = name =>
-  isLocalStorageAvailable()
-    ? localStorage.removeItem(name)
-    : store.dispatch(removeDataFromRedux({ name }));
+export const removeData = name => localStorage.removeItem(name);
 
 export const sendMessage = msg => {
```

Comment:
we can remove this - the definition and the usage. It was idea for hosted solution